src/zdaemon/tests/tests.py

```diff
@@ -79,7 +79,7 @@ def dont_hang_when_program_doesnt_start():
     >>> write('conf',
     ... '''
     ... <runner>
-    ...   program sleep
+    ...   program sleep .1
```

Comment:
On Python 3.5 `governor()` gets called 2 seconds after the spawn() of the quickly-exitting process, so it thinks there's nothing bad.

I currently suspect the following code in `runforever()`:
```python
            try:
                r, w, x = select.select(r, w, x, timeout)
            except select.error as err:
                if err.args[0] != errno.EINTR:
                    raise
                r = w = x = []
```

Python 3.5 handles EINTR internally instead of letting applications see it.